carbonmark/lib/createUrls.ts

```diff
@@ -1,11 +1,20 @@
+import { NetworkParam } from ".generated/carbonmark-api-sdk/types";
+import { urls } from "@klimadao/lib/constants";
 import { Project, TokenPrice } from "lib/types/carbonmark.types";
 
 type ProjectData = {
   key: Project["key"];
   vintage: Project["vintage"];
+  serialization?: Project["serialization"] | null;
+};
+
+export const createProjectLink = (projectData: ProjectData) => {
+  if (projectData.key.startsWith("ICR") && projectData.serialization) {
+    return `/projects/${projectData.serialization}`;
+  } else {
+    return `/projects/${projectData.key}-${projectData.vintage}`;
```

Comment:
this works if `projectData.key.startsWith("ICR") == true` and there is no `projectData.serialization` ?